docs/application/dotnet/guides/nui/nui-components/CheckBox.md

```diff
@@ -1,77 +1,205 @@
 
-1. Create CheckBox using the default constructor:
+To create checkbox using property, follow these steps:
 
-    ```cs
-    CheckBox utilityCheckBox = new CheckBox();
-    ```
+1. Create an instance of a `CheckBox` class using the default constructor:
+   ```cs
+   CheckBox _checkBox = new CheckBox();
+   ```
 
-2. Apply style to the CheckBox:
+2. Set the checkbox properties:
+   ```cs
+   // Path to the directory with images
+   string _URL = Tizen.Applications.Application.Current.DirectoryInfo.Resource + "images/";
 
-    ```cs
-    ButtonStyle utilityStyle = new ButtonStyle
+   _checkBox.Size = new Size(200, 200);
+   _checkBox.ParentOrigin = ParentOrigin.Center;
+   _checkBox.PositionUsesPivotPoint = true;
+   _checkBox.PivotPoint = PivotPoint.Center;
+
+   // Set the icon images for different checkbox states 
+   StringSelector _iconURL = new StringSelector()
+   {
+      Normal           = _URL + "blue.png",
+      Selected         = _URL + "blue_checked.png",
+      Pressed          = _URL + "red.png",
+      Disabled         = _URL + "green.png",
+      DisabledSelected = _URL + "yellow.png"
+   };
+   _checkBox.IconURLSelector = _iconURL;
+   _checkBox.Icon.Size = new Size2D(160,160);
+   _checkBox.BackgroundColor = new Color(0.57f, 0.7f, 1.0f, 0.8f);
+
+   // CheckBox initial state set to be selected
+   _checkBox.IsSelected = true;
+   ```
+
+   To set the absolute path of the images that are used, the `Tizen.Applications.Application.Current.DirectoryInfo.Resource` path is used. For more information, see [Class Application](https://samsung.github.io/TizenFX/latest/api/Tizen.Applications.Application.html) and [Class DirectoryInfo](https://samsung.github.io/TizenFX/latest/api/Tizen.Applications.DirectoryInfo.html).
+
+3. Add checkbox to the view:
+   ```cs
+   _rootView.Add(_checkBox);
+   ```
+
+The following output is generated when the checkbox is created using properties:
+
+![CheckBox_prop](./media/CheckBox_prop.gif) 
+
+A checkbox selection can be disabled by adding the following code:
+   ```cs
+   // CheckBox can not be selected 
+   _checkBox.IsSelectable = false;
+   ```
+
+To disable the checkbox completely use the following code:
+   ```cs
+   // CheckBox is disabled
+   _checkBox.IsEnabled = false;
+   ```
+This sets the checkbox state to `Disabled` or `DisabledSelected` depending on the `IsSelected` value. The preceding options change the checkbox appearance as follows:
+
+| ![CheckBox_dis](./media/CheckBox_dis.gif)                   | ![CheckBox_dis](./media/CheckBox_yellow.gif)             | ![CheckBox_dis](./media/CheckBox_green.gif)               |
+|:-----------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------------------:|
+| ```IsSelectable``` set to `false`<br>`IsSelected` set to `true` | `IsEnabled` set to `false`<br>`IsSelected` set to `true` | `IsEnabled` set to `false`<br>`IsSelected` set to `false` |
```

Comment:
In this page, I'm a little bit confused below point. 

1. In my understanding, in the first case the button can be clicked, but button value is not changeable (`IsSelectable` = `false`), and the button value is "checked" (`IsSelected` = `true`).
If so, isn't the button always in the checked state even how many times it is clicked?
In the image, first it is in checked state, and changed to unchecked state at the click. After that it keeps unchecked state even it is clicked again many times. So I'm a little bit confused. If I understood something wrong, would you please let me know?

2. In the second and the third case, the buttons are disabled, 
so even it is clicked the red image doesn't appear, means not clickable.
The second button is checked, so it is in yellow. (DisabledSelected = _URL + "yellow.png").
The third button is not checked, so it is in green. (Disabled         = _URL + "green.png",)

   However, in the second image, the yellow seems overlayed on blue, so it looks like yellowish green. Is it possible to strengthen the yellow in the image?

Reply:
please check now.
1. After reconsideration, I don't think it's a good idea to use IsSelectable=false and IsSelected=true. the behaviour is not obvious (it depends on the order of commands). I changed IsSelected to false.  Please let me know whether it is clear now.
2. I've changed the opacity. is there anything else that requires clarification?

Reply:
Thanks for the update. I read the updated parts and now they are clear to me. I hope @annie-samsung also can check the updated parts and let us know the opinion.